solidity/package.json

```diff
@@ -56,15 +53,14 @@
     "bn.js": "^4.11.8",
     "chai": "^4.2.0",
     "eslint": "^5.16.0",
-    "eslint-config-keep": "git+https://github.com/keep-network/eslint-config-keep.git#0.2.0",
-    "eslint-config-prettier": "^6.10.0",
```

Comment:
I believe we need this guy:
```
> eslint .


Oops! Something went wrong! :(

ESLint: 5.16.0.
ESLint couldn't find the plugin "eslint-plugin-prettier". This can happen for a couple different reasons:
```

Reply:
This should be pulled in by eslint-config-keep. Did you `npm i`?

Reply:
I'm afraid it had never worked for me if I didn't add `eslint-config-prettier` explicitly. I upgraded npm today and boom it works!